src/Dmishh/Bundle/SettingsBundle/Manager/CachedManager.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Dmishh\Bundle\SettingsBundle\Manager;
+
+use Dmishh\Bundle\SettingsBundle\Entity\SettingsOwnerInterface;
+use Doctrine\Common\Cache\CacheProvider;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+/**
+ * @author Tobias Nyholm
+ */
+class CachedManager implements SettingsManagerInterface
+{
+    const PREFIX = 'dmishh_o{%s}_v{%s}';
```

Comment:
Sure, This is actually the second prefix. The proper one is configured when you define your cache service. So, `{%s}_{%s}` would be enough. But since we are lazy and don't want to define too many cache services I've added the `dmishh_`